opm/material/fluidmatrixinteractions/EclMaterialLawManager.hpp

```diff
@@ -982,13 +1014,18 @@ class EclMaterialLawManager
     std::vector<Opm::EclEpsScalingPointsInfo<Scalar>> unscaledEpsInfo_;
     OilWaterScalingInfoVector oilWaterScaledEpsInfoDrainage_;
 
-    Opm::EclMultiplexerApproach threePhaseApproach_;
+    GasOilScalingPointsVector gasOilUnscaledPointsVector_;
```

Comment:
If we depend on storing these data (which are cell data, right?), we must probably also fix the parallel case, when the properties gets distributed, to also distribute these data. Otherwise I would expect this to break flow_mpi and also flow_ebos with MPI (eventually).